source/faq.txt

```diff
@@ -183,6 +183,14 @@ The error message consists of multiple parts:
      can use tools like ``openssl s_client`` to debug TLS/SSL-related
      certificate problems.
 
+.. _csharp-faq-linq-vs-builder:
+
+Should I Use LINQ or Builder Classes When Querying for Documents?
+-----------------------------------------------------------------
+
+If you define your documents using C# classes, use LINQ. If you use the
+``BsonDocument`` class to represent your documents, use builders. 
```

Comment:
I see that this was the example used in the Jira ticket, but I don't think this is necessarily true. In our docs we define our documents with C# classes in both the LINQ and the Builders pages. I think the difference between LINQ and Builders is more based on your use case and the type of query you're building. We also decided when creating the C# docs that we wouldn't push users towards using BsonDocuments, but to advocate for POCOs instead in most/all cases.

I think we should check with the engineering team on whether we should add this comparison section, and if so, get their opinion on when to use one or the other.

Reply:
Good shout, will reach out to the C# team!